files/apt/sources.list.j2

```diff
@@ -0,0 +1,20 @@
+# The configuration is generated by template
+# Please add additional sources in /etc/apt/sources.list.d
+
+{% for mirror_url in MIRROR_URLS.split(',') %}
+deb {{ mirror_url }} {{ DISTRIBUTION }} main contrib non-free
+deb-src {{ mirror_url }} {{ DISTRIBUTION }} main contrib non-free
+deb {{ mirror_url }} {{ DISTRIBUTION }}-updates main contrib non-free
+deb-src {{ mirror_url }} {{ DISTRIBUTION }}-updates main contrib non-free
+deb {{ mirror_url }} {{ DISTRIBUTION }}-backports main contrib non-free
+#deb-src {{ mirror_url }} {{ DISTRIBUTION }}-backports main contrib non-free
+{% endfor %}
+{% for mirror_url in MIRROR_SECURITY_URLS.split(',') %}
+{% if DISTRIBUTION == 'stretch' %}
+deb {{ mirror_url }} {{ DISTRIBUTION }}/updates main contrib non-free
+deb-src {{ mirror_url }} {{ DISTRIBUTION }}/updates main contrib non-free
+{% else %}
+deb {{ mirror_url }} {{ DISTRIBUTION }}-security main contrib non-free
+deb-src {{ mirror_url }} {{ DISTRIBUTION }}-security main contrib non-free
+{% endif %}
```

Comment:
Format was changed in bullseye: https://www.debian.org/releases/stable/amd64/release-notes/ch-information.en.html#security-archive
Buster also has format {{ DISTRIBUTION }}/updates for security mirrors.

Reply:
@kv-y , fixed, thanks for you comment.